lib_eio_windows/test/dune

```diff
@@ -1,5 +1,7 @@
 (test
-  (name test)
+  (name test)  ; Main test suite
+  (libraries alcotest kcas eio.mock eio_windows)
```

Comment:
These changes are not necessary (I don't think the comment is too important and shifting the order only adds extra things to review).

Reply:
Sorry about the addition of extra things to review. Would remove it.